pango-view/viewer-render.h

```diff
@@ -32,6 +32,13 @@ typedef enum {
   HINT_FULL
 } HintMode;
 
+typedef struct {
+  double x;
+  double y;
+  double width;
+  double height;
+} LayoutExtents;
```

Comment:
You can use `cairo_rectangle_t` instead.

Reply:
Thanks, this is now fixed.